src/basic/pager.c

```diff
@@ -62,9 +63,17 @@ int pager_open(bool no_pager, bool jump_to_end) {
         if (!pager)
                 pager = getenv("PAGER");
 
-        /* If the pager is explicitly turned off, honour it */
-        if (pager && STR_IN_SET(pager, "", "cat"))
-                return 0;
+        if (pager) {
+                pager_args = strv_split(pager, WHITESPACE);
+                if (!pager_args)
+                        return -ENOMEM;
+
+                /* If the pager is explicitly turned off, honour it */
+                if (strv_isempty(pager_args) || streq(pager_args[0], "cat")) {
```

Comment:
Somewhat nitpick-y... But should this check that pager_args[1] is NULL for the second case with "cat"? Otherwise `PAGER='cat -v'` or similar will also disable it, while that might have been done with a purpose (a questionable one, but still.)

Reply:
Yeah, it is questionable. `cat -v` is not a pager.

Reply:
We have `strv_equal`, so what about just doing
```C
if (strv_isempty(pager_args) || strv_equal(pager_args, STRV_MAKE("cat"))
   …
```
?